view/frontend/web/js/pagseguro.js

```diff
@@ -1,340 +0,0 @@
-/**
```

Comment:
Deleting this file will affect creditCard payment, and many other functionalities of the module. Am I wrong?

Reply:
@r-martins Sorry I have the wrong files commit to git now resolved issue. please check and let me know if any concern.